content/videos/teachable-machine/1-image-classification/index.json

```diff
@@ -0,0 +1,105 @@
+  "videoId": "kwcillcWOg0",
+  "date": "2019-11-7",
+  "languages": ["JavaScript", "p5.js", "ml5.js"],
+  "topics": ["machine learning", "ml5","image classification"],
+  "canContribute": false,
+  "timestamps": [
+    { "time": "0:32", "title": "Teachable Machine 2.0" },
+    { "time": "1:06", "title": "Training an Image Classification model" },
+    { "time": "3:11", "title": "Train model" },
+    { "time": "3:21", "title": "High-level overview" },
+    { "time": "5:10", "title": "Save your data" },
+    { "time": "6:58", "title": "Train the model" },
+    { "time": "7:30", "title": "Save the model" },
+    { "time": "8:57", "title": "Coding" },
+    { "time": "10:31", "title": "Step 1 Load the model" },
+    { "time": "11:19", "title": "Step 2 Classify" },
+    { "time": "12:44", "title": "Step 3 Get the classification" },
+    { "time": "14:48", "title": "Step 4 Draw the label" },
+    { "time": "18:34", "title": "The sound classifier" },
+    { "time": "19:19", "title": "Train a model that can be a game controller" }
+  ],
+  "codeExamples": [
+    {
+      "title": "Code Template",
+      "description": "Template code for the Teachable Machine.",
+      "urls":{
+        "p5": "https://editor.p5js.org/codingtrain/sketches/aRN4PS2Nm"
+      } 
+    },
+    {
+      "title": "Final Code",
+      "description": "Final code of the Teachable Machine.",
+      "urls":{
+        "p5": "https://editor.p5js.org/codingtrain/sketches/PoZXqbu4v"
+      } 
+    }
+  ],
+  "groupLinks": [
+    {
+      "title": "References",
+      "links": [
+        {
+          "title": "Teachable Machine",
+          "url": "https://teachablemachine.withgoogle.com/",
+          "description": "A web-based tool that makes creating machine learning models fast, easy, and accessible to everyone."
+        },
+        {
+          "icon": "💾",
+          "title": "Tensor Flow",
+          "url": "https://www.tensorflow.org/js",
+          "description": "A library for machine learning in JavaScript."
+        },
+        {
+          "icon": "💻",
+          "title": "ml5.js",
+          "url": "https://ml5js.org/",
+          "description": "ml5 page with documentation and supporting material."
+        },
+        {
+          "icon": "💻",
+          "title": "Image-Net",
+          "url": "http://image-net.org/index",
+          "description": "ImageNet is an image database organized according to the WordNet hierarchy."
+        },
+        {
+          "icon": "📓",
+          "title": "MobileNets: Efficient Convolutional Neural Networks for Mobile Vision Applications",
+          "url": "https://arxiv.org/abs/1704.04861",
+          "description": "Academic article about MobileNets."
+        }
+      ]
+    },
+    {
+      "title": "Videos",
+      "links": [
+        {
+          "icon": "🚂",
+          "title": "ml5.js: Transfer Learning with Feature Extractor",
+          "url": "ml5js-beginners-guide/ml5/2-transfer-learning/1-transfer-learning",
```

Comment:
Both these internal video links should be prefixed with "/tracks/" to work properly

Reply:
it does work without as well because they are both inside the tracks path already. I have made the change as you suggested though

Reply:
Thanks. They weren't working on the preview link for me. They don't link to a video in the same track, they link to the "ml5js-beginners-guide" track.